components/Meme.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
```

Comment:
Since you're using the latest version, consider not importing react. It's not necessary because React does it for you. This'd be sufficient: 
```suggestion
import { useEffect } from "react";
```